cmd/account/mgmt/account-assign.go

```diff
@@ -63,16 +65,18 @@ func newCmdAccountAssign(streams genericclioptions.IOStreams, flags *genericclio
 		Use:               "assign",
 		Short:             "Assign account to user",
 		DisableAutoGenTag: true,
-		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(ops.complete(cmd, args))
-			cmdutil.CheckErr(ops.run())
-
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := ops.complete(cmd, args)
```

Comment:
Might be useful to at least mention in the commit message that `CheckErr` was removed and the parameter was changed to `RunE` for testability purposes.

Reply:
We can probably re-introduce the `CheckErr` wrapper here now, right?